agent/containermetadata/parse_metadata.go

```diff
@@ -27,10 +28,11 @@ import (
 // available prior to container creation
 // Since we accept incomplete metadata fields, we should not return
 // errors here and handle them at this or the above stage.
-func (manager *metadataManager) parseMetadataAtContainerCreate(taskARN string, containerName string) Metadata {
+func (manager *metadataManager) parseMetadataAtContainerCreate(task *api.Task, containerName string) Metadata {
+	taskDefinitionRevision := parseTaskDefinitionRevision(task, containerName)
 	return Metadata{
 		cluster:              manager.cluster,
-		taskMetadata:         TaskMetadata{containerName: containerName, taskARN: taskARN},
+		taskMetadata:         TaskMetadata{containerName: containerName, taskARN: task.Arn, taskDefinitionFamily: task.Family, taskDefinitionRevision: taskDefinitionRevision},
```

Comment:
minor: can you split these into multiple lines?

Reply:
Yeah. I couldn't figure out what the line length convention for this repo is. I'll do something reasonable.